packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.controller.js

```diff
@@ -79,12 +81,14 @@ export default class ScrubbingCenterController extends AbstractCursorDatagridCon
   }
 
   selectService() {
+    this.selectedIp = null;
+    this.emptyList = false;
```

Comment:
name is confusing, I'm expected to have `[]` as value
```suggestion
    this.isEmpty = false;
```